proto/viam/app/cloudslam/v1/cloud_slam.proto

```diff
@@ -102,6 +102,8 @@ message UpdateMappingSessionMetadataByIDRequest {
   string end_status = 2;
```

Comment:
We ideally wanted this to be `enum end_status` (not string), but since that'd be a breaking change, we'll use string for now and just add `error_msg`